easybuild/easyconfigs/t/TensorFlow/TensorFlow-0.11.0-CrayGNU-2016.11-cuda-8.0-Python-3.5.2.eb

```diff
@@ -0,0 +1,58 @@
+# @author: marcelsc
+
+easyblock = "CmdCp"
+
+name = 'TensorFlow'
+version = '0.11.0'
+pyver = '3.5.2'
+cudaversion = '8.0'
+versionsuffix = '-cuda-%s-Python-%s' % (cudaversion, pyver)
+
+homepage = 'https://www.tensorflow.org/'
+description = """An open-source software library for Machine Intelligence."""
+
+toolchain = {'name': 'CrayGNU', 'version': '2016.11'}
+
+dependencies = [
+    ('Python','3.5.2'),
+    ('cuDNN', '5.1', '', True),
+    ('SWIG', '3.0.10', '-Python-%(pyver)s'),
+    ('Bazel', '0.4.0'),
+    ('protobuf', '3.0.2', '-Python-%(pyver)s'),
+]
+
+sources = ['v%(version)s.tar.gz']
+source_urls = ['https://github.com/tensorflow/tensorflow/archive/']
+
+patches = [('/apps/daint/UES/6.0.UP02/sandbox-ds/soures/tensorflow-v%(version)s-cscs.patch')]
```

Comment:
please don't use the absolute path for patches, you should include them on the PR and EB will search for them automatically.